src/components/FileCard/index.js

```diff
@@ -0,0 +1,172 @@
+import { FiMoreVertical } from 'react-icons/fi'
+import { RiMoreFill } from 'react-icons/ri'
+import styled from 'styled-components'
+
+const CardContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap:${props => props.small ? 6 : 29}px;
+  width: 159px;
+  height: ${props => props.small ? '79px' : '133px'};
+  background: #FFFFFF;
+  box-shadow: 0px 4px 16px rgba(58, 58, 58, 0.07);
+  border-radius: 12px;
+  .file_options{
+    display: ${props => props.small ? 'none' : 'flex'};
+    position: absolute;
+    right: 13px;
+    top: 16px;
+  }
+  .file_graphic{
+    display: flex;
+    align-items: center;
+    gap: 35px;
+    margin-top: ${props => props.small ? 13 : 20}px;;
+    margin-left: 12px;
+    .square_shadow{
+      position: relative;
+      width: 30px;
+      height: 26px;
+      background: ${props => props.small ? '#FF9F00' : '#FFD999'};
+      .square{
+        position: absolute;
+        top: -5px;
+        right: -5px;
+        width: 30px;
+        height: 26px;
+        background: ${props => props.small ? 'transparent' : '#FF9F00'};
+      }
+    };
+    .file_changes{
+      position: relative;
+      width: 18px;
+      height: 18px;
+    }
+  };
+  .file_info{
+    display: flex;
+    flex-direction: column;
+    margin-left: 12px;
+    gap: ${props => props.small ? 2 : 10}px;;
+    h2{
+      font-weight: normal;
+      font-size: 12px;
+      line-height: 14px;
+      color: #343951;
+      margin: 0px;
+    }
+    p{
+      font-size: 8px;
+      line-height: 9px;
+      letter-spacing: 0.05em;
+      color: #858A9D;
+      margin: 0px;
+    }
+  };
+`;
+
+const CircleInfo = styled.div`
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  right: ${props => props.index==1 ? 0 : (props.index-1)*-12}px;
+  width: 18px;
+  height: 18px;
+  background: #FFFFFF;
+  border: 1px solid ${props => props.color ? props.color : '#BEC0C8'};
+  border-radius: 9px;
+  z-index: ${props => props.index ? props.index : 1};
+  p{
+    font-weight: normal;
+    font-size: 10px;
+    line-height: 9px;
+    color: #343951;
+    margin: 0px;
+  }
+`
+
+function FolderCard ({ files, created, size, title }){
+  return(
+    <CardContainer small={size}>
+      <FiMoreVertical className='file_options' color='#C4C4C4'/>
+      <div className='file_graphic'>
+        <div className='square_shadow'>
+          <div className='square'/>
+        </div>
+        <div className='file_changes'>
+          {files.map( (file, index) => {
+              if(index < 3){
+                return(
+                  <CircleInfo key={index} index={index} color={file.color}>
+                    {(files.length>1 && index==2) && <p>+3</p>}
+                  </CircleInfo>
+                )
+              }
+            }
+          )}
```

Comment:
We could change this to a guard clause: 

```
              if(index > 2) return null
                return(
                  <CircleInfo key={index} index={index} color={file.color}>
                   // I don't think you need to specify that files.length is > 1 since index has to be == 2
                    {index==2 && <p>+3</p>}
                  </CircleInfo>
                )
            }
          )}
```